manifest.json

```diff
@@ -18,7 +18,6 @@
         }
     },
     "permissions": [
-        "https://maps.googleapis.com/*",    
```

Comment:
The reason you did not seem to have to add anything here is that NW.js does not use `manifest.json`. The Chrome Web App does, and for it to load the map you'll have to add `https://api.openstreetmap.org/` in here.

The good news is that once this is done, the map loads and displays just fine in the Chrome Web App. Good work!

Reply:
I tested the chrome app but is clear that I did something wrong... I will add it.

Reply:
Added...